styles/mixins/patterns/_molecules.scss

```diff
@@ -59,3 +59,33 @@ $-space: var.$layout-space;
     @include layout.vertical-rhythm($-space * 0.5);
   }
 }
+
+// A responsive sized and positioned container for modal content
+@mixin modal {
+  // The modal container is positioned horizontally- and vertically-centered
+  @include layout.fixed-centered;
+
+  // For narrower viewports/default:
+  // - positioning: centered vertically and horizontally in the viewport
+  // - sizing: fills up most of the available horizontal space
+  width: 90vw;
+  max-width: 90vw;
+  max-height: 90vh;
+
+  // For wider viewports:
+  // - positioning: keep horizontal centering but bring top nearer to top
+  //     of viewport.
+  // - sizing: Set a min-width to make sure the modal isn't too dinky. Set
+  //     a max-width to make sure it isn't too ginormously wide on wide screens.
+  @media screen and (min-width: 48rem) {
```

Comment:
Bah, I had it in an earlier comment here but then deleted it and neglected to re-add: this `48rem` magic is a rem-translation of the `768px` tablet breakpoint from the client. I will make sure to add commenting.